tests/compiler/ternary.release.wat

```diff
@@ -10,6 +10,7 @@
  (data (i32.const 1036) "\1c")
  (data (i32.const 1048) "\02\00\00\00\02\00\00\00a")
  (table $0 1 1 funcref)
+ (elem $0 (i32.const 1))
```

Comment:
Any idea what's going on here? Seems strange that this affects just a few release tests.

Reply:
No idea. Perhaps a some regression in Binaryen?

Reply:
debug fixture contain this: https://github.com/AssemblyScript/assemblyscript/blob/c7eafd336d843bc6bd2a0796613f731955cd8f63/tests/compiler/std/array-access.debug.wat#L23